src/frontends/tensorflow/docs/contribute.md

```diff
@@ -0,0 +1,71 @@
+# How to implement support of a new TensorFlow operation
+
+The conversion of a TensorFlow operation requires either one pass using [Loaders](../src/op) or two transformation passes
+using [Loaders](../src/op) and [Internal Transformation](../src/helper_transforms). It is often sufficient to use only [Loaders](../src/op) for the conversion.
+Two transformation passes are used when a TensorFlow operation cannot be mapped into a sub-graph of the OpenVINO opset,
+and the conversion depends on the succeeding operations in the graph.
+
+## One transformation pass using Loader
+
+Most TensorFlow operations can be converted by one transformation pass using `Loader`.
+The dictionary of `Loaders` is placed in the [op_table.cpp](../src/op_table.cpp) file and loaders in the [op](../src/op) directory:
+
+```
+const std::map<std::string, CreatorFunction> get_supported_ops() {
+    return {
+        {"Abs", translate_unary_op<opset8::Abs>},
+        {"Acos", translate_unary_op<opset8::Acos>},
+        {"Acosh", translate_unary_op<opset8::Acosh>},
+        ...
+    };
+};
+```
+
+Here is an example of `Loader` for TensorFlow `Einsum` operation:
+
+https://github.com/rkazants/openvino/blob/rkazants/tf_fe_docs_82500/src/frontends/tensorflow/src/op/einsum.cpp#L15-L28
```

Comment:
This link should be changed to original openvino repo

Reply:
Done.